### PR TITLE
Removed Device Not Applied DataType, Added new Modes

### PR DESCRIPTION
**Pull request by RohitKandimalla**
MAT-2410
## Description

- Added new Modes 'class' and 'interpretation'.
- Removed DataType "Device, Not Applied" and its associated attributes.
## JIRA Ticket

[https://jira.cms.gov/browse/MAT-2410](MAT-2410)
## Checklist

- [ ] If UI changes have been made, google WAVE plug-in has been executed to ensure no 508 issues were introduced.
- [ ] Tests are included and test edge cases.
- [ ] Tests have been run locally and pass.

## Screenshots (if appropriate)
- [ ] None applicable

### CODE REVIEW

File: src/main/resources/SimplifiedAttributePatterns.xml [hunk 1]
```diff
@@ -1,229 +1,239 @@
...
+    <attribute attribName="class">
+        <mode mode="Nullable" />
+        <mode mode="ValueSets" />
+        <mode mode="Codes" />
+    </attribute>
+    <attribute attribName="interpretation">
+        <mode mode="Nullable" />
+        <mode mode="ValueSets" />
+        <mode mode="Codes" />
+    </attribute>
```
Comment (jkotanchik-SB) [inline src/main/resources/SimplifiedAttributePatterns.xml:238]: 👍  Nice catch!

**Review by jkotanchik-SB — APPROVED** [review]
